src/sentry/static/sentry/app/views/settings/components/settingsProjectItem.jsx

```diff
@@ -0,0 +1,93 @@
+import styled from 'react-emotion';
+import React from 'react';
+
+import createReactClass from 'create-react-class';
+import theme from '../../../utils/theme';
+
+import {update} from '../../../actionCreators/projects';
+import ApiMixin from '../../../mixins/apiMixin';
+import Tooltip from '../../../components/tooltip';
+import Link from '../../../components/link';
+import ProjectLabel from '../../../components/projectLabel';
+import SentryTypes from '../../../proptypes';
+
+const Header = styled('h5')`
+  margin: 0;
+`;
+
+const InlineButton = styled('button')`
+  color: ${p => p.theme.gray3};
+  border: none;
```

Comment:
do you know if there's a better unstyled button?

Reply:
Try: `<Button borderless>`

Reply:
had trouble sizing this and went back to InlineButton 😬